core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -34,6 +34,13 @@ $(call force,CFG_GIC,y)
 $(call force,CFG_PL011,y)
 $(call force,CFG_PM_STUBS,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+CFG_CORE_TPM_EVENT_LOG ?= y
```

Comment:
This code is not platform-specific so I'd recommend `CFG_CORE_TPM_EVENT_LOG ?= n` in `mk/config.mk` (with some short documentation).